settings.gradle.kts

```diff
@@ -7,6 +7,19 @@ pluginManagement {
     includeBuild("build-logic")
 }
 
+buildscript {
+    configurations.all {
+        resolutionStrategy {
+            force("org.jetbrains.kotlin:kotlin-bom:1.9.24")
+        }
+    }
+}
+
+plugins {
+    id("com.autonomousapps.build-health") version "2.17.0"
```

Comment:
To get the full benefit of it, please add the respective buildHealth command to the build workflow and add the output to the log via "dependency.analysis.print.build.health=true" or alternatively save the report file somewhere accessable.

Reply:
Done